parking/src/Ticket.py

```diff
@@ -0,0 +1,86 @@
+import time
+from parking.models import TariffPlan, carDataDetails
+	
+class Ticket:
+    ''' Class used for update/create tariff plans and for
+	    generating tickets with fare details for cars at exit.'''
+		
+    def __init__(self, location) :
+        '''Constructor method'''
+        self.outTime = time.time()
+        car = self.__findVehical(location)
+        self.carno  = car.carno
+        self.inTime = float(car.inTime)
+        self.location = car.location
+        self.tariff    = car.tariff_plan
+        self.ID        = self.__generateTicketID()
+        self.amount   = None
+        self.tariffMinutesDict = {"Hourly" : 3600, 
+                                  "Daily" : 86400 }
+		
+    def __findVehical(self,location):
+        ''' This method searches and returns in the database for car
+		    based on location details provided'''
+        return carDataDetails.objects.get(location=location)
+
+    def __generateTicketID(self):
+        ''' Generates a random id for a ticket'''
+        import random
+        ID = random.randint(1,1000)
+        return ID
+    def __getTariffAmount(self):
+        ''' Method returns the cost and freetime for tariff plan selected.'''
+        tariff = TariffPlan.objects.get(plan=self.tariff)
+        return (tariff.cost,tariff.freetime)
+
+    def __calAmount(self):
```

Comment:
don't hesitate to express want you want in function names, `def _calculate_amount()` its a good idea. 

_ as very personal opinion I think long names in private methods aren't that bad, while short and expressive names are very good for interfaces/public methods._